CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ endif()
 set (GTSAM_VERSION_MAJOR 4)
 set (GTSAM_VERSION_MINOR 2)
 set (GTSAM_VERSION_PATCH 0)
-set (GTSAM_PRERELEASE_VERSION "a0")
+set (GTSAM_PRERELEASE_VERSION "a1")
```

Comment:
How come this is in this PR? Did you merge in develop into the wrong branch?

Reply:
If so, merge developed into the base branch as well, so we only see the relevant diff here.

Reply:
Okay done!